app/templates/components/archive-project-milestone-list-item.hbs

```diff
@@ -27,7 +27,7 @@
       {{/each}}
       <br>
       <small class="display-block">
-        Recommendation submitted&nbsp;{{~moment-format this.anyCommunityBoardDispositionDateReceived "MM/D/YYYY"~}}
+        Recommendation submitted&nbsp;<DateDisplay @date={{this.anyCommunityBoardDispositionDateReceived}} @outputFormat='MM/D/YYYY' />
```

Comment:
Awesome, thanks for figuring this out @godfreyyeung. Question to understand: why is it necessary to specify @outputFormat="MM/D/YYYY" when that's the preferred format? I figured that was being set as the default outputFormat in line 13 of the new component, so you'd only need to specific the @outputFormat in the template if it was overriding the default of "MM/D/YYYY".

Reply:
Oh yeah good catch! that's true in this case it's not necessary. I was mass-transforming each `moment-format` to `DisplayDate`. I'll update